internal/cmd/shell/shell.go

```diff
@@ -188,10 +196,10 @@ second argument:
 		"", "Local address to bind and listen for connections. By default the proxy binds to 127.0.0.1 with a random port.")
 	cmd.PersistentFlags().StringVar(&flags.remoteAddr, "remote-addr", "",
 		"PlanetScale Database remote network address. By default the remote address is populated automatically from the PlanetScale API.")
-	cmd.PersistentFlags().BoolVar(&flags.readOnly, "read-only",
-		false, "Connect to database in read only mode.")
+	cmd.PersistentFlags().StringVar(&flags.role, "role",
+		"reader", "allowed values are : reader, writer, readwriter, admin, default is reader")
```

Comment:
Would like to see a text message that describes the flag. Such as: 

`Role defines the access level, allowed values are : reader, writer, readwriter, admin. By default it is reader.`